### PR TITLE
move formatter job to GitHub Actions

### PR DESCRIPTION
**Pull request by SimeonEhrig**

### CODE REVIEW

File: .github/workflows/formatter.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: formatter
+on: [push, pull_request]
```
Comment (szabo137) [inline .github/workflows/formatter.yaml:2]: is it really necessary to trigger the formatter on *push*? I mean, the trigger on *pull_request* should run the job again, if you push into a pending PR, isn't it?

Comment (SimeonEhrig) [reply .github/workflows/formatter.yaml:2]: Actually, no. But I thought it would not hurt.

Comment (SimeonEhrig) [reply .github/workflows/formatter.yaml]: I changed it to `pull_request` only.

**Review by szabo137 — APPROVED** [review]
All conversations are resolved. Ready for merging!

**Pull request merged**